doc/example/example_ordinal.ipynb

```diff
@@ -62,9 +62,417 @@
+       "4                       lin            normal         ordinal   \n",
+       "5                       lin            normal         ordinal   \n",
+       "6                       lin            normal         ordinal   \n",
+       "7                       lin            normal         ordinal   \n",
+       "8                       lin            normal         ordinal   \n",
+       "9                       lin            normal         ordinal   \n",
+       "10                      lin            normal         ordinal   \n",
+       "11                      lin            normal         ordinal   \n",
+       "12                      lin            normal         ordinal   \n",
+       "13                      lin            normal         ordinal   \n",
+       "14                      lin            normal         ordinal   \n",
+       "15                      lin            normal         ordinal   \n",
+       "16                      lin            normal         ordinal   \n",
+       "17                      lin            normal         ordinal   \n",
+       "\n",
+       "    measurementCategory  \n",
+       "0                     2  \n",
+       "1                     2  \n",
+       "2                     3  \n",
+       "3                     3  \n",
+       "4                     2  \n",
+       "5                     1  \n",
+       "6                     1  \n",
+       "7                     1  \n",
+       "8                     1  \n",
+       "9                     1  \n",
+       "10                    1  \n",
+       "11                    1  \n",
+       "12                    2  \n",
+       "13                    2  \n",
+       "14                    3  \n",
+       "15                    3  \n",
+       "16                    3  \n",
+       "17                    3  "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from pandas import option_context\n",
+    "\n",
+    "with option_context('display.max_colwidth', 400):\n",
+    "    display(petab_problem.measurement_df)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For ordinal measurements, the `measurement` column will be ignored.\n",
+    "\n",
+    "Measurements with a larger category number will be constrained to be higher in the ordering. Multiple measurements can be assigned to the same category. If this is done, these measurements will be treated as indistinguishable. \n",
+    "\n",
+    "#### Note on inclusion of additional data types:\n",
+    "It is possible to include observables with different types of data to the same `petab_problem`. Refer to `./example_nonlinear_monotone.ipynb` and `./example_censored.ipynb` for details on integration of other data types. If the `measurementType` column is left empty for all measurements of an observable, the observable will be treated as quantitative."
```

Comment:
I would like to nicely link the notebooks of different data types to each other. I haven't been able to do that. Need a bit of help.

Reply:
What exactly would you like to link to? The notebook on github? The output on readthedocs? A link to open a new notebook in your jupyter session?

Reply:
I would like to link this notebook to the ordinal and nonlinear monotone notebook (at `./example_nonlinear_monotone.ipynb`). And the same in those notebooks to the other two.

Reply:
Oh, to the readthedocs

Reply:
> Oh, to the readthedocs. I guess I can give the web link to that, 

You should use relative links  (e.g. `example_nonlinear_monotone.html`), otherwise it will not link to the documentation of the same release you are browsing. But then again, those links will not work when you are executing the notebook locally, because the html files won't exist. Linking to `example_nonlinear_monotone.ipynb` will work locally, but won't work as intended on RTD.

> but I think there's some nicer internal way of doing that within the docs? 

Not really, unfortunately.

Reply:
Yeah, I don't mind if it's not possible to access it locally. I would just like it to be accessible in the readthedocs. I would say that's nicer, as the first place to search for examples is usually in the online docs. 
So in this case I should link it to the HTML file? But with which path exactly? Should it be `/doc/_build/html/example/example_nonlinear_monotone.html`?

Reply:
just `example_nonlinear_monotone.html` should suffice.

Reply:
`example_nonlinear_monotone.html` didn't work. But `example_nonlinear_monotone.ipynb` seems to work as a link in the docs and locally for notebooks (If I'm not missing something...). :D